pallets/system/src/tests.rs

```diff
@@ -663,41 +664,86 @@ fn assert_runtime_updated_digest(num: usize) {
 	);
 }
 
-/*
-#[test]
-fn set_code_with_real_wasm_blob() {
-	let executor = new_native_executor();
-	let mut ext = new_test_ext();
-	ext.register_extension(sp_core::traits::ReadRuntimeVersionExt::new(executor));
-	ext.execute_with(|| {
-		System::set_block_number(1);
-		System::set_code(
-			RawOrigin::Root.into(),
-			substrate_test_runtime_client::runtime::wasm_binary_unwrap().to_vec(),
-		)
-		.unwrap();
-
-		assert_eq!(System::events(), vec![EventRecord {
-			phase: Phase::Initialization,
-			event: SysEvent::CodeUpdated.into(),
-			topics: vec![],
-		}],);
-	});
-}
-
-#[test]
-fn runtime_upgraded_with_set_storage() {
-	let executor = new_native_executor();
-	let mut ext = new_test_ext();
-	ext.register_extension(sp_core::traits::ReadRuntimeVersionExt::new(executor));
-	ext.execute_with(|| {
-		System::set_storage(RawOrigin::Root.into(), vec![(
-			well_known_keys::CODE.to_vec(),
-			substrate_test_runtime_client::runtime::wasm_binary_unwrap().to_vec(),
-		)])
-		.unwrap();
-	});
-}*/
+// #[test]
+// fn set_code_with_real_wasm_blob() {
+// 	let executor = substrate_test_runtime_client::new_native_or_wasm_executor();
+// 	let mut ext = new_test_ext();
+// 	ext.register_extension(sp_core::traits::ReadRuntimeVersionExt::new(executor));
+// 	ext.execute_with(|| {
+// 		System::set_block_number(1);
+// 		System::set_code(
+// 			RawOrigin::Root.into(),
+// 			substrate_test_runtime_client::runtime::wasm_binary_unwrap().to_vec(),
+// 		)
+// 		.unwrap();
+
+// 		assert_eq!(
+// 			System::events(),
+// 			vec![EventRecord {
+// 				phase: Phase::Initialization,
+// 				event: SysEvent::CodeUpdated.into(),
+// 				topics: vec![],
+// 			}],
+// 		);
+// 	});
+// }
+
+// #[test]
+// fn set_code_via_authorization_works() {
```

Comment:
Any reason for commenting out this test?

Reply:
@ToufeeqP like the two older tests nearby, they use `substrate_test_runtime_client` which has an imcompatible deps so we would have to fork it. which is not really required. You can try if you want.